identifiers/country-jp/house_of_councilors.csv

```diff
@@ -0,0 +1,46 @@
+id,name
+ocd-division/country:jp/at_large_district:aichi,Aichi

```

Comment:
Hmm, in other countries this is modelled as:

`ocd-division/country:jp/prefecture:23/ed:at-large`

I think this makes more sense, since the at-large district only makes sense in the context of the prefecture.

Reply:
Can you suggest how to represent the combined prefectures in the suggested format?

For example: 
`ocd-division/country:jp/at_large_district:tokushima-kochi`
`ocd-division/country:jp/at_large_district:tottori-shimane`